sale_properties_easy_creation/mrp_property_formula.py

```diff
@@ -0,0 +1,30 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+#
+#    Copyright (C) 2014-15 Agile Business Group sagl
+#    (<http://www.agilebg.com>)
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as published
+#    by the Free Software Foundation, either version 3 of the License, or
+#    (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+
+from openerp.osv import orm, fields
+
+
+class MrpPropertyFormula(orm.Model):
+    _name = 'mrp.property.formula'
+    _columns = {
+        'name': fields.char('Name', size=128),
+        'formula_text': fields.text('Formula'),
```

Comment:
I think that the code to evaluate the formula should be in a helper method in this class  to avoid duplication. 

Otherwise we are going to have to watch for things like https://github.com/OCA/sale-workflow/pull/146/files#diff-ef08d502d2e08be1274eeab2f6f78114R64 in many places.